store/src/main/java/com/nytimes/android/external/store/base/impl/RealInternalStore.java

```diff
@@ -313,15 +312,7 @@ void notifySubscribers(Parsed data) {
     @Nonnull
     @Override
     public Observable<Parsed> stream(@Nonnull Key key) {
-
-        Observable<Parsed> stream = subject.asObservable();
-
-        //If nothing was emitted through the subject yet, start stream with get() value
-        if (!subject.hasValue()) {
-            return stream.startWith(get(key));
-        }
-
-        return stream;
+        return subject.asObservable().startWith(get(key));
```

Comment:
No need for `asObservable()` as startWith is already an `Observable` and no longer a `Subject`